src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1691,6 +1691,41 @@ object FunctionRegistry {
     registerMethod("min", (x: Any, y: Any) => ord.min(x, y))
     registerMethod("max", (x: Any, y: Any) => ord.max(x, y))
 
+
+    registerMethod("argmin", (a: IndexedSeq[Any]) => {
+      if (a.isEmpty)
+        null: java.lang.Integer
+      else {
+        var min = a(0)
+        var minIndex = 0
+        var i = 1
+        while (i < a.length) {
```

Comment:
Is the reason you didn't combine the two functions with a unique flag because of speed?
You could have done 
```
if (count == 1 || !unique)
    mi
else
   null
```

Reply:
Performance is a concern, yeah.